client/src/components/App/Setup.js

```diff
@@ -71,10 +71,22 @@ class Setup extends Component {
     const errorMessage = this.validate();
     // Redirect to home if already registered
     if (registered) {
-      return <Redirect to="/" />;
+      setTimeout(() => { window.location.href = '/'; }, 5000);
     }
     return (
       <Card classes={css.setup} title="Registrering for generalforsamling">
+        <Dialog
+          hideCloseSymbol
+          onClose={() => { window.location.href = '/'; }}
+          title={'En feil har oppstått'}
+          visible={registered}
+        >
+          <h3>Du er allerede registrert</h3>
```

Comment:
Doesn't help much when Dialog doesn't pass it on...

https://github.com/dotkom/super-duper-fiesta/blob/aa2e536b848b01858dfdc1c02c8a95e4b9a8cbae/client/src/components/Dialog.js#L21-L25